docs/io.cozy.files.md

```diff
@@ -168,6 +168,63 @@ The `io.cozy.files` doctype has [the standard `cozyMetadata`](https://docs.cozy.
 }
 ```
 
+### Shortcuts
+
+Shortcuts (or hyperlinks) are a special type of files. They are served
+with the `application/shortcut` mime-type, they have a `.url` extension, and
+respect the [URL file
+format](http://www.lyberty.com/encyc/articles/tech/dot_url_format_-_an_unofficial_guide.html).
+
+They can have `metadata`:
+
+- `sharing-status` will be set if the link can be used to open a sharing. 2
```

Comment:
How do you know ? It could have been opened in desktop, and then not catched

Reply:
I'm not decided, but an option would be that the sharing link goes to the recipient cozy, and then redirects to the sharer cozy instance. The product team would like to show this information in Drive Web, so I'm trying to do that.